File: Wrp/cs/src/com/amivoice/wrp/WrpListener.cs
namespace com.amivoice.wrp {

public interface WrpListener {
	void utteranceStarted(int startTime);
	void utteranceEnded(int endTime);
	void resultCreated();
	void resultUpdated(string result);
	void resultFinalized(string result);
	void eventNotified(int eventId, string eventMessage);
	void TRACE(string message);
}

} // namespace

File: Wrp/java/src/com/amivoice/wrp/WrpListener.java
package com.amivoice.wrp;

public interface WrpListener {
	void utteranceStarted(int startTime);
	void utteranceEnded(int endTime);
	void resultCreated();
	void resultUpdated(String result);
	void resultFinalized(String result);
	void eventNotified(int eventId, String eventMessage);
	void TRACE(String message);
}
